modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionLocalServiceImpl.java

```diff
@@ -123,7 +123,8 @@ public ListTypeDefinition deleteListTypeDefinition(
 	@Indexable(type = IndexableType.REINDEX)
 	@Override
 	public ListTypeDefinition updateListTypeDefinition(
-			long listTypeDefinitionId, Map<Locale, String> nameMap)
+			long listTypeDefinitionId, Map<Locale, String> nameMap,
+			String externalReferenceCode)
```

Comment:
@murilostodolni @mateussgn `String externalReferenceCode` must come before `listTypeDefinitionId`, following the same order from the `service.xml`